sdk/core/amqp-common/src/requestResponseLink.ts

```diff
@@ -122,25 +123,58 @@ export class RequestResponseLink implements ReqResLink {
           request.message_id = generate_uuid();
         }
 
+        const rejectOnAbort = () => {
+          const address = this.receiver.address || "address";
+          const requestName = options!.requestName || "requestName";
+          const desc: string =
+            `[${this.connection.id}] The request "${requestName}" ` +
```

Comment:
Connection Id should only be used for logging. The actual error message that is returned to the user should not contain the connectionId. The other place where we return a rejected a promise in this file, doesnt use the connection id in the error message. I am ok following the same for now

Reply:
Ah I see, I took the overall pattern from the sender/receiver error descriptions, but those only get logged, not thrown (they use a different error when thrown). Will update.